services/notifier.js

```diff
@@ -17,5 +18,9 @@ module.exports = {
         for (platform in rooms) {
             notifiers[platform].sendMessage(rooms[platform], from, msg, color);
         }
+    },
+
+    sendSlackPersonalMessage: function(userEmail, from, msg, color) {
```

Comment:
I found it useful to not generate the 'handledname' for each call. Instead the SlackPersonalMessage does it so If the 'handled' name would change in the future, we will change this function only. And it is more clear when reading the code.